client/src/dbus/api/mod.rs

```diff
@@ -13,6 +13,8 @@ impl Unlockable for &zbus::zvariant::OwnedObjectPath {}
 mod collection;
 mod item;
 mod prompt;
+pub mod properties;
+#[cfg(not(feature = "unstable"))]
```

Comment:
This change is not needed, would drop the whole commit

Reply:
Done, thanks!